ethstore/src/accounts_dir/disk.rs

```diff
@@ -191,7 +207,7 @@ impl<T> DiskDirectory<T> where T: KeyFileManager {
 					.map_err(Into::into)
 					.and_then(|file| self.key_manager.read(filename, file))
 					.map_err(|err| {
-						warn!("Invalid key file: {:?} ({})", path, err);
+						eprintln!("Invalid key file: {:?} ({})", path, err);
```

Comment:
hrm, why `eprintln` here instead of rustlog? I believe we want it to be rustlog

Reply:
because otherwise, the error message is swallowed by `.ok()` 4 lines below, and if you are running `ethstore/cli` user has no idea what went wrong since there is no logging enabled...

Reply:
Ok, I see but not the `ok()'s` fault IMO. The method is badly designed for your purpose because it only propagates the error if the  `Vec<PathBuf>` fails.

/cc  @sorpaas @debris are we concerned by the performance penalty that `eprintln!` comes with in this context? I'm not exactly sure when this is code is used in the `core client` but I guess it is when we sign transactions and similar?

Reply:
I haven't expected that keyfiles may be read upon every sign (ECC) ops... some dive into parity code has not confirmed this, but if it is the case then I should restore `warn!`.

My first version of this enables logging in `ethstore/cli`. This is also acceptable to me.